src/routes/oauth2/middleware.js

```diff
@@ -29,10 +32,21 @@ module.exports = {
     const requestJWT = req.jwt;
     try {
       if (requestJWT) {
-        const apiResponse = await req.axios.post(sessionPath, {
-          request: req.jwt,
-          client_id: req.session.authParams.client_id,
-        });
+        const headers = createPersonalDataHeaders(
+          `${req.app.get("API.BASE_URL")}${req.app.get("API.PATHS.SESSION")}`,
+          req,
+        );
```

Comment:
I see there's no existing tests to update but what do you think about adding one for this API call so we can cover the new functionality?

Reply:
unit tests already exist around axios request. But your comment gives me the idea to add the new cloudfront headers into existing unit tests.
Like that we are sure that is working.